app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -142,8 +149,30 @@ val APP_SETTINGS_DEFAULT = AppSettings(
     usePrivateTabs = false,
     secureWindow = false,
     blurNSFW = true,
+    saveSearchHistory = true,
+)
+
+@Entity
+data class SearchHistory(
+    @PrimaryKey @ColumnInfo(name = "text") val text: String,
+    @ColumnInfo(name = "timestamp") val timestamp: Long,
```

Comment:
This column isn't necessary, the inserts are going to be in order anyway by the primary key.